src/core/control/tools/StrokeHandler.h

```diff
@@ -72,6 +72,8 @@ class StrokeHandler: public InputHandler {
     SnapToGridInputHandler snappingHandler;
 
 private:
+    void createMask();
```

Comment:
This needs a doc comment. Also, I think this should be renamed to some more descriptive like `createStrokeRenderBuffer`. I get that technically we're applying a cairo mask to the page render buffer surface, but that obscures the real intention (we draw the incoming points to a buffer that we can use for full repaint).